src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -228,7 +229,14 @@ public void parseCommand_updateSessionRemark() throws Exception {
         assertTrue(parser.parseCommand(UpdateSessionRemarkCommand.COMMAND_WORD
                 + whiteSpace + PREFIX_SESSION + sessionNumber
                 + whiteSpace + PREFIX_SESSION_REMARK + sessionRemark) instanceof UpdateSessionRemarkCommand);
+    }
 
+    @Test
```

Comment:
hmm, i see alot of the same `String whiteSpace = " "` being used. Might want to change it to a global constant?

Reply:
Sounds good! Added the final String for this file and replaced the " " strings within each test to use `WHITESPACE` instead